format/format.go

```diff
@@ -1108,13 +1108,29 @@ func (w *writer) objectWriter() entryWriter {
 		entry := x.([2]*ast.Term)
 		comments = w.writeTerm(entry[0], comments)
 		w.write(": ")
+
+		call, isCall := entry[1].Value.(ast.Call)
+		if isCall && ast.Or.Ref().Equal(call[0].Value) && entry[1].Location.Text[0] == 40 { // Starts with "("
```

Comment:
This should also cover when the call is spread over multiple lines, right?
E.g.: 
```rego
union_object := {"response": (
  declare3 | declare4
)}
```
Let's add that case to the test.

Reply:
Yes. Added a test case